build.gradle

```diff
@@ -108,6 +108,24 @@ void setupAllTargetsWithDefaultSourceSets(Project project, Boolean isLinuxArm32H
                 iosCommonMain.dependsOn nativeCommonMain
                 iosCommonTest.dependsOn nativeCommonTest
 
+                watchosArm32Main.dependsOn iosCommonMain
+                watchosArm32Test.dependsOn iosCommonTest
+
+                watchosArm64Main.dependsOn iosCommonMain
+                watchosArm64Test.dependsOn iosCommonTest
+
+                watchosSimMain.dependsOn iosCommonMain
+                watchosSimTest.dependsOn iosCommonTest
+
+                tvosArm64Main.dependsOn iosCommonMain
+                tvosArm64Test.dependsOn iosCommonTest
+
+                tvosSimMain.dependsOn iosCommonMain
+                tvosSimTest.dependsOn iosCommonTest
+
+                macosX64Main.dependsOn iosCommonMain
```

Comment:
Why macos main depends on iosCommon? Maybe rename `iosCommonXxx` to `darwinCommonXxx`?